python/pyarrow/_s3fs.pyx

```diff
@@ -81,9 +81,9 @@ cdef class S3FileSystem(FileSystem):
     cdef:
         CS3FileSystem* s3fs
 
-    def __init__(self, *, access_key=None, secret_key=None, anonymous=False,
-                 region=None, scheme=None, endpoint_override=None,
-                 bint background_writes=True):
+    def __init__(self, *, access_key=None, secret_key=None, session_token=None,
```

Comment:
You must add `session_token` in the docstring above.

Reply:
updated docstring accordingly, thanks for catching that.